### ISSUE TITLE
Builds are failing due to missing git.apache.org/thrift.git dependency

### ISSUE THREAD

**Issue by Smiley73** [KohlsTechnology/eunomia#90]
## What happens?
Travis builds are failing due to `git.apache.org/thrift.git` not being available.

## What were you expecting to happen?
Builds complete without problems

## Steps to reproduce: 
Kick off a build through travis.

## Any errors, stacktrace, logs? 
```
go: finding git.apache.org/thrift.git v0.12.0
go: git.apache.org/thrift.git@v0.12.0: unknown revision v0.12.0
go: error loading module requirements
```
## Environment:
Travis CI

**Comment by seanmalloy** [comment]
I can reproduce this without using Travis CI with the below commands.

```
$ rm -rf vendor
$ rm -rf $GOPATH/pkg/mod
$ export GO111MODULE=on
$ go mod vendor
...
go: git.apache.org/thrift.git@v0.12.0: unknown revision v0.12.0
go: error loading module requirements
```

**Comment by seanmalloy** [comment]
This is fixed by #91
